Lib/test/test_unittest/test_async_case.py

```diff
@@ -434,6 +434,30 @@ async def cleanup(self, fut):
         test.doCleanups()
         self.assertEqual(events, ['asyncSetUp', 'test', 'cleanup'])
 
+    def test_setup_get_event_loop(self):
+        # See https://github.com/python/cpython/issues/95736
+        events = []
+        class TestCase1(unittest.IsolatedAsyncioTestCase):
+            def setUp(self):
+                self.loop = asyncio.get_event_loop_policy().get_event_loop()
+
+            async def test_demo1(self):
+                events.append('called demo1')
+
+
+        class TestCase2(unittest.IsolatedAsyncioTestCase):
```

Comment:
What is the difference between TestCase1 and TestCase2? Why do you need both of them.

Reply:
One breaks the environment, the other detects the breakage. The plan now is to manually break the environment then test that

Reply:
Simplified now